Cargo.toml

```diff
@@ -120,6 +120,7 @@ convert-case = ["convert_case"]
 preserve_order = ["indexmap", "toml?/preserve_order", "serde_json?/preserve_order", "ron?/indexmap"]
 async = ["async-trait"]
 toml = ["dep:toml"]
+path_to_error = ["serde_path_to_error"]
```

Comment:
I would be interested in seeing a proposal for what it would look like to have this built-in

Reply:
Do you mean without any external dependency? I think it is doable, but probably we will copy & paste from the other repository.
Or maybe I miss-interpreted the point. Can you elaborate more?
thanks

Reply:
Without it being optional and baked into the existing error reporting

Reply:
remove optional